pure3.md

```diff
@@ -0,0 +1,282 @@
+in-vehicle communication bandwidth is not constrained. As before, the in-toto
+verification process used can be either the full workflow or the stop-gap
+option, as this choice does not affect individual ECUs, but only the setup as a
+whole.
+
+Note that the stop-gap option can be employed in conjunction with the solutions
+for the other constraints. A vehicle may simultaneously have no ECUs capable of
+full in-toto verification _and_ limited bandwidth for in-vehicle communication.
+Similarly, a vehicle may have no ECUs capable of full in-toto verification _and_
+some of these ECUs may be incapable of any in-toto verification at all.
+
+## Security Analysis
+
+Scudo is an extension for Uptane that adds in-toto's software supply chain
+security properties. Therefore, its threat model is a combination of the threat
+models of Uptane for the delivery of the software artifacts, and in-toto for the
+development of the image as it passes through the software supply chain.
+
+## Backwards Compatibility
+
+Implementers of Scudo should always consider backwards compatibility. In itself,
+this PURE should not introduce any backwards incompatibility to existing Uptane
+deployments. However, introducing in-toto into the mix can complicate matters.
+Therefore, neither an Uptane nor in-toto implementation should make
+backward-incompatible changes to how signatures are generated. This will ensure
+that previous package managers are able to continue to install new packages.
+Note that Uptane can otherwise be used to safely rotate the keys for the entire
+system, including those using different key types, key sizes, signature schemes,
+and cryptographic hashes. Any backwards incompatible changes that are
+unavoidable should be handled with a clear plan, perhaps making use of the
+semantics introduced in TAP-14.
+
+## Copyright
+
+This document has been placed in the public domain.
+
+## Acknowledgements
+
+This PURE was inspired by two in-toto Enhancements, ITE-2 and ITE-3, that
+describe how TUF, Uptane's parent project, and in-toto can be combined.
+
+## References
+
+* [in-toto Specification](https://github.com/in-toto/docs/blob/master/in-toto-spec.md)
+* [Uptane v2.0.0 Standard](https://uptane.github.io/papers/uptane-standard.2.0.0.html)
+* [Scudo Whitepaper](https://uptane.github.io/papers/scudo-whitepaper.pdf)
+* [ITE-2](https://github.com/in-toto/ITE/blob/master/ITE/2/README.adoc)
+* [ITE-3](https://github.com/in-toto/ITE/blob/master/ITE/3/README.adoc)
+* [Sonatype's 2022 State of the Software Supply Chain report](https://www.sonatype.com/state-of-the-software-supply-chain/introduction)
+* [2020 Sunburst attack](https://www.cynet.com/attack-techniques-hands-on/sunburst-backdoor-c2-communication-protocol/)
+* [Datadog's Verification of TUF and in-toto metadata](https://github.com/DataDog/integrations-core/blob/df7d87fe2888800bccb49c01a8cc5afa0f016e62/datadog_checks_downloader/datadog_checks/downloader/download.py#L282)
```

Comment:
I think promoting this more up front as a POC is important

Reply:
I added a prototype section that talks about this.

Reply:
cc @trishankatdatadog